platform/linux-generic/arch/common/odp_time_cpu.c

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2013-2018, Linaro Limited
+ * Copyright (c) 2020-2023, Nokia
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#include <odp/api/time_types.h>
+
+#include <odp/api/abi/time_cpu.h>
+#include <odp/api/abi/time_cpu_inlines.h>
+
+#include <odp_debug_internal.h>
+#include <odp_init_internal.h>
+
+#include <inttypes.h>
+#include <stdint.h>
+#include <string.h>
+
+#include <odp/visibility_begin.h>
+
+_odp_time_global_t _odp_time_glob;
+
+#include <odp/visibility_end.h>
+
+int _odp_time_init_global(void)
+{
+	_odp_time_global_t *global = &_odp_time_glob;
+
+	memset(global, 0, sizeof(_odp_time_global_t));
+
+	if (!_odp_time_cpu_global_freq_is_const())
+		_ODP_ERR("HW time frequency may change\n");
```

Comment:
This warning could be CAPS to be more visible. Also, could comment the reasoning a bit: "Freq should be constant based on CPU ISA version, but could not confirm from CPU flag" or something similar.

Reply:
Fixed in V2.